WooCommerce/Classes/ViewRelated/Products/Edit Product/Downloads/Downloadable FIle List/ProductDownloadListViewController.swift

```diff
@@ -0,0 +1,139 @@
+        onCompletion = completion
+        super.init(nibName: type(of: self).nibName, bundle: nil)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureAddButton()
+        configureAddButtonSeparator()
+        registerTableViewCells()
+        configureTableView()
+        configureNavigationBar()
+        handleSwipeBackGesture()
+    }
+}
+
+// MARK: - View Configuration
+//
+private extension ProductDownloadListViewController {
+    func registerTableViewCells() {
+        tableView.register(ImageAndTitleAndTextTableViewCell.loadNib(), forCellReuseIdentifier: ImageAndTitleAndTextTableViewCell.reuseIdentifier)
+    }
+
+    func configureAddButton() {
+        addButton.setTitle(NSLocalizedString("Add File", comment: "Action to add downloadable file on the Product Downloadable Files screen"), for: .normal)
+        addButton.addTarget(self, action: #selector(addButtonTapped), for: .touchUpInside)
+        addButton.applySecondaryButtonStyle()
+    }
+
+    func configureAddButtonSeparator() {
+        addButtonSeparator.backgroundColor = .systemColor(.separator)
+    }
+
+    func configureTableView() {
+        view.backgroundColor = .listBackground
+        tableView.backgroundColor = .listBackground
+        tableView.dataSource = self
+        tableView.delegate = self
+        tableView.removeLastCellSeparator()
+        tableView.dragInteractionEnabled = true
+        tableView.dragDelegate = self
+        tableView.dropDelegate = self
+        tableView.reloadData()
+    }
+
+    func configureNavigationBar() {
+        configureTitle()
+        configureRightButtons()
+    }
+
+    func configureTitle() {
+        title = NSLocalizedString("Downloadable Files",
+                                  comment: "Edit product downloadable files screen - Screen title")
+    }
+
+    func configureRightButtons() {
+        var rightBarButtonItems = [UIBarButtonItem]()
+
+        let moreBarButton: UIBarButtonItem = {
+            let button = UIBarButtonItem(image: .moreImage,
+                                         style: .plain,
+                                         target: self,
+                                         action: #selector(moreButtonTapped))
+            button.accessibilityTraits = .button
+            button.accessibilityLabel = NSLocalizedString("Update products' downloadable files",
+                                                          comment: "The action to update products' downloadable files settings")
+            return button
+        }()
+        rightBarButtonItems.append(moreBarButton)
+
+        let doneButtonTitle = NSLocalizedString("Done",
+                                               comment: "Edit product downloadable files screen - button title to add downloadable files selection")
```

Comment:
The description seems wrong to me. It should be something like this:
```suggestion
                                               comment: "Edit product downloadable files screen - button title to apply changes to downloadable files selection")
```

Reply:
Changed